spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/websocket/servlet/WebSocketMessagingAutoConfigurationTests.java

```diff
@@ -138,13 +137,10 @@ void customizedConverterTypesMatchDefaultConverterTypes() {
 		return customizedConverters;
 	}
 
-	@SuppressWarnings("unchecked")
 	private List<MessageConverter> getDefaultConverters() {
 		DelegatingWebSocketMessageBrokerConfiguration configuration = new DelegatingWebSocketMessageBrokerConfiguration();
-		// We shouldn't usually call this method directly since it's on a non-proxy config
```

Comment:
I was unsure about this one. I didn't get why we shouldn't call the method directly inside tests.

Reply:
looks safe to me

Reply:
My first guess was that the method hasn’t always been public, but that’s not the case.

The change to using reflection was made in [this polishing commit](https://github.com/spring-projects/spring-boot/commit/68aec8b6ee5e3c51c12f7a17fe4b0cef1709f908) by @philwebb. My second guess is that the change was made to avoid [our tooling](https://github.com/wilkinsona/spring-boot-development-eclipse-plugin) reporting an error about a direct call to an unproxied `@Bean` method.

Reply:
@wilkinsona Should I keep the proposed change or do you want me to revert this particular one?